Source/Core/Color.js

```diff
@@ -632,6 +632,27 @@ Color.prototype.toCssColorString = function () {
   return "rgba(" + red + "," + green + "," + blue + "," + this.alpha + ")";
 };
 
+/**
+ * Creates a string containing CSS hex string color value for this color.
```

Comment:
There should be some kind of note that this ignores the alpha value of the color.